HPA with Stormforger (#33)
thestormforge/examples: hpa/stormforger/README.md

# Voting Webapp with HPA optimized using Stormforger load test

The goal of this recipe is to optimize the HPA used for the voting webapp.
In this experiment the load test is performed by [Stormforger](stormforger.com). This allows to generate much heavier load on the website and show how carefully tuning HPA along with the deployed application allows to handle such traffic.

## Deploy the voting webapp with ingress

> In this case the load test resides outside of the cluster. The voting webapp needs to expose an endpoint for in&out cluster traffic.

Reviewer: ```suggestion
Because the load test resides outside of the cluster, the voting webapp needs to be exposed publicly.
```

Run:
`kustomize build application | kubectl apply -f -`

> Once the ip address for the ingress is available you can test the website by accessing the IP address in a web browser or using curl.

Reviewer: ```suggestion
Once the IP address for the ingress is available you can test the website by accessing the IP address in a web browser or using curl.
Write
```
